scanner/src/main/java/edu/ucr/cs/riple/scanner/Config.java

```diff
@@ -53,6 +56,26 @@ public interface Config {
   @Nonnull
   Path getOutputDirectory();
 
+  Context getContext();
+
+  class Context {
+    public final Set<MethodInfo> methodIs;
```

Comment:
Also, no idea why we are collecting this set of method info records here, as part of `Context`, but either way this isn't a very descriptive field name. Maybe `methodInfoRecords`?

Reply:
This is used to not to increment unique id for an already visited method. It is currently `visitedMethods` please let me know if you still think `methodInfoRecords` is more descriptive.

Reply:
I am fine with `visitedMethods`